flowauth/frontend/src/UserAdminDetails.js

```diff
@@ -65,6 +66,24 @@ class UserAdminDetails extends React.Component {
     var state = {
       [name]: event.target.value
     };
+    if (name === "name") {
+      var letters = /^[A-Za-z_]+$/;
+      let username = event.target.value;
+      console.log(username);
+      if (username.match(letters) && username.length > 5) {
+        state = Object.assign(state, {
+          usernameError: ""
+        });
+      } else if (username.length <= 5) {
+        state = Object.assign(state, {
+          usernameError: "Username should be more than 6 charactor"
+        });
+      } else {
+        state = Object.assign(state, {
+          usernameError: "Please use only letters and _"
+        });
+      }
+    }
     if (name === "password") {
       var passStrength = zxcvbn(event.target.value);
```

Comment:
One thing that might be nice here is to use the fact that zxcvbn will actually give you the reason for the score being low, in the feedback property of this object as password hint text.